vnext/Chakra/ChakraHelpers.cpp

```diff
@@ -49,6 +49,7 @@ bool fwrite(const T& val, FILE* file) noexcept
   return fwrite(&val, 1, file) == 1;
 }
 
+#if !defined(CHAKRACOREUWP)
```

Comment:
I would like to suggest making the define `CHAKRACORE_UWP`.

Reply:
whoops. did catch this comment before the automerge. I'll send a follow-up renaming this